src/test/java/io/vertx/httpproxy/impl/ParseUtilsTest.java

```diff
@@ -0,0 +1,57 @@
+package io.vertx.httpproxy.impl;
+
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+
+import static junit.framework.TestCase.assertEquals;
+
+@RunWith(VertxUnitRunner.class)
```

Comment:
```suggestion
```

Why do you need the extension? It seems to me the test is pretty straightforward: it uses date time formats and that's it.